editions/tw5.com/tiddlers/filters/limit.tid

```diff
@@ -8,12 +8,15 @@ purpose: select the first or last <<.place n>> input titles
 
 <$macrocall $name=".operator-def"
 input="a [[selection of titles|Title Selection]]"
+suffix="optional, an offset"
+suffixName="o"
 parameter="an integer, defaulting to 0"
 paramName="n"
 output="the first <<.place n>> input titles"
-negationOutput="the last <<.place n>> input titles"
```

Comment:
The code still seems to process the negation `!`, why aren't we documenting the behaviour?